### ISSUE TITLE
[Bug] no status light

### ISSUE THREAD

**Issue by luweijun1992** [benphelps/homepage#1157]
### Description

no status light

### Steps to reproduce

no status light

### homepage version

v0.6.10

### Installation method

Docker

### Configuration

```yaml
- Portainer:
        icon: portainer.png
        href: https://192.168.248.10:9443
        description: The analytics platform for all your metrics
        server: my-docker
        container: portainer
        widget:
            type: portainer
            url: https://192.168.248.10:9443
            env: 2
            key: password

---

my-docker:
    socket: /var/run/docker.sock
```

### Other

I have been unable to see the status light.
![image](https://user-images.githubusercontent.com/58923704/228484690-563e5f6e-0a8f-42c3-9cd3-2d943f95cd11.png)

### Before submitting, I have made sure to

- [X] Check [the documentation](https://gethomepage.dev/)
- [X] Follow [the troubleshooting guide](https://gethomepage.dev/en/more/troubleshooting/) (please include output above if applicable).
- [X] Search [existing issues](https://github.com/benphelps/homepage/search?q=&type=issues) and [discussions](https://github.com/benphelps/homepage/search?q=&type=discussions).

**Comment by AlexanderBottlang** [comment]
**tl;dr** - You are seeing the "status light" already, it's the green **"RUNNING"** text in the upper right corner.

I first was confused what you meant by "status light", so I consulted the documentation. There is only one full match when searching; in the chapter [Docker Integration](https://gethomepage.dev/en/configs/services/#docker-integration).

The documnetation and screenshots are just outdated. As a user of homepage for the last couple months now, I can remember the status lights pictured in the screenshots. Now we have some text explaining further, instead of just a colored dot.

So you can rest assured, that the status light is already visible. The documentation and screenshots just need an update.
